msg/sensor_combined.msg

```diff
@@ -1,73 +1,24 @@
-# Definition of the sensor_combined uORB topic.
-
-int32 MAGNETOMETER_MODE_NORMAL = 0
-int32 MAGNETOMETER_MODE_POSITIVE_BIAS = 1
-int32 MAGNETOMETER_MODE_NEGATIVE_BIAS = 2
-
-uint32 SENSOR_PRIO_MIN = 0
-uint32 SENSOR_PRIO_VERY_LOW = 25
-uint32 SENSOR_PRIO_LOW = 50
-uint32 SENSOR_PRIO_DEFAULT = 75
-uint32 SENSOR_PRIO_HIGH = 100
-uint32 SENSOR_PRIO_VERY_HIGH = 125
-uint32 SENSOR_PRIO_MAX = 255
-
-# Sensor readings in raw and SI-unit form.
 #
-# These values are read from the sensors. Raw values are in sensor-specific units,
-# the scaled values are in SI-units, as visible from the ending of the variable
-# or the comments. The use of the SI fields is in general advised, as these fields
-# are scaled and offset-compensated where possible and do not change with board
-# revisions and sensor updates.
+# Sensor readings in SI-unit form.
 #
-# Actual data, this is specific to the type of data which is stored in this struct
-# A line containing L0GME will be added by the Python logging code generator to the logged dataset.
+# These fields are scaled and offset-compensated where possible and do not
+# change with board revisions and sensor updates.
 #
-# NOTE: Ordering of fields optimized to align to 32 bit / 4 bytes Change with consideration only
 
-uint64[3] gyro_timestamp			# Gyro timestamps
-int16[9] gyro_raw			# Raw sensor values of angular velocity
-float32[9] gyro_rad_s			# Angular velocity in radian per seconds
-uint32[3] gyro_priority			# Sensor priority
-float32[9] gyro_integral_rad		# delta angle in radians
-uint64[3] gyro_integral_dt			# delta time for gyro integral in us
-uint32[3] gyro_errcount			# Error counter for gyro 0
-float32[3] gyro_temp			# Temperature of gyro 0
+int32 RELATIVE_TIMESTAMP_INVALID = 2147483647 # (0x7fffffff) If one of the relative timestamp is set to this value, it means the associated sensor values are invalid
 
-int16[9] accelerometer_raw		# Raw acceleration in NED body frame
-float32[9] accelerometer_m_s2		# Acceleration in NED body frame, in m/s^2
-float32[9] accelerometer_integral_m_s		# velocity in NED body frame, in m/s^2
-uint64[3] accelerometer_integral_dt		# delta time for accel integral in us
-int16[3] accelerometer_mode			# Accelerometer measurement mode
-float32[3] accelerometer_range_m_s2		# Accelerometer measurement range in m/s^2
-uint64[3] accelerometer_timestamp	# Accelerometer timestamp
-uint32[3] accelerometer_priority	# Sensor priority
-uint32[3] accelerometer_errcount	# Error counter for accel 0
-float32[3] accelerometer_temp		# Temperature of accel 0
+# gyro timstamp is equal to the timestamp of the message
+float32[3] gyro_integral_rad		# delta angle in the NED body frame in rad in the integration time frame
+uint32 gyro_integral_dt			# delta time for gyro integral in us
```

Comment:
I would argue this should be a `float32` in seconds. I'm not completely happy that you need to do `/ 1.e6f;` everywhere below.